docs(README): add images missing in the doc (#198)
ekino/docker-buildbox: README.md

Amazon Linux based image containing Systemd for service management in docker container.

### Azure

Contains Azure Cli + Terraform.

### Chrome

> Contains Chromium browser (installed with Node:10-alpine).

Reviewer: ```suggestion
Contains Chromium browser.
```
to make future updates easier

Reviewer 2: Or we could juste write this ?
```suggestion
Contains Chromium browser and the latest Node LTS.
```
That way the user knows Node is available and we don't have to edit it either.

### Scoutsuite

Contains ScoutSuite cloud scanner.

### Serverless

Contains node Serverless module with python3.

### SonarQube Scanner

Contains SonarQube Scanner and CI Helper.

### Terraform

> Contains Terraform (installed with aws-cli).

Reviewer: ```suggestion
Contains Terraform and AWS Cli.
```
to follow same pattern than other descriptions
